ppr-ui/src/components/mhrRegistration/HomeOwners/FractionalOwnership.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <div id="mhr-home-ownership">
+    <label class="generic-label">Fractional Ownership</label>
+    <div v-if="isReadOnly">
+      <p data-test-id="readonly-interest-info">Interest: {{ fractionalInterest }}</p>
+    </div>
+    <div v-else>
+      <p class="mt-3 mb-6">
+        Enter the interest type and fraction of the total ownership owned by Group 1. For example,
+        if there are four owner groups, this group could have 1/4 ownership.
+      </p>
+      <v-text-field
+        :id="`interest-type-group-${groupId}`"
+        label="Interest Type (Optional)"
+        v-model="fractionalInfo.interest"
+        :data-test-id="`interest-type-field-group-${groupId}`"
+      />
+      <div class="owner-fractions">
+        <v-text-field
+          :id="`fraction-amount-group-${groupId}`"
+          label="Amount Owned by this Group"
+          filled
+          v-model="fractionalInfo.interestNumerator"
+          :rules="fractionalAmountRules"
+          :data-test-id="`fraction-amount-field-group-${groupId}`"
+        />
+        <span> </span>
```

Comment:
Intentional span?

Reply:
Sorry, should've made a comment. This is for that diagonal line (aka fraction divider "/") between two fields. Used css to create it.